src/app/components/configs-list/configs-list.component.scss

```diff
@@ -0,0 +1,56 @@
+.configs-list_wrapper {
+  width: 240px;
+  position: absolute;
+  top: 85px;
+  left: 5px;
+  border-radius: 5px;
+  background-color: white;
+  box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2), 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 1px 5px 0 rgba(0, 0, 0, 0.12);
+  .configs-list_container {
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    flex-wrap: wrap;
+
+    .title {
+      margin: 0 5px
+    }
+    .config {
+        max-width: 70px;
+        min-width: 70px;
+        padding: 6px;
+        height: 60px;
+        margin: 5px;
+        border: 1px solid #ddd;
+        /* box-shadow: 0 3px 1px -2px rgba(0,0,0,.2), 0 2px 2px 0 rgba(0,0,0,.14), 0 1px 5px 0 rgba(0,0,0,.12); */
+        border-top: 4px solid;
+        text-overflow: ellipsis;
+        width: 110px;
+        white-space: nowrap;
+        overflow: hidden;
+        display: inline-block;
+        vertical-align: middle;
+        border-radius: 5px;
+        cursor: pointer;
+        text-decoration: unset;
+        color: black;
+    }
+    .config:hover {
+      background-color: rgba(100, 100, 100, 0.1);
+    }
+    .config:active {
+      background-color: rgba(100, 100, 100, 0.2);
+    }
+
+  }
```

Comment:
Instead of rewrite a rule like `.config:hover` or `.config:active`, you can add a sub rule inside the `.config` rule, like this :

```
.config{
    ...

    &:hover{
        ...
    }

    &:active{
        ...
    }
}
```